system setup

ajsilveira opened the pull request:
## Description
Provide a brief description of the PR's purpose here.

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - [ ] TODO 1

## Questions
- [ ] Question1

## Status
- [ ] Ready to go

Commit:
add rocsalt system setup

Commit:
fix doc

Commit:
fix for considering neutral ligands

Commit:
some fixes

hannahbrucemacdonald commented on rocsalt/rocsalt_system.py:
```diff
@@ -0,0 +1,534 @@
+            self.ligands.append(Ligand(resname, total_charge, 'off', ligand_indices))
+
+        # Determine counterions and dummy atoms required if ligands are charged
+        charges = [ligand.charge for ligand in self.ligands]
```
Do we need to assert that the two ligands have the same charge?

jchodera commented:
Would be great to get this merged in today!
Are there other files to bring in, like your example input files?

Commit:
add host-guest parameter files
add yaml script
handle files by paths

Commit:
change logic for determining ions/dummies in alchemical protocol
resnames are arbitrarily assigned
specify ligands in yaml file by their resnames
complex and solvent are class Phase objects

Commit:
use yank functionalities for determining alchemical counterions
strip unused ions from complex/solvent ParmEd structures

Commit:
remove data folder
add host-guests and kinase-inhibitors examples

Commit:
add ligands annealing in solvent

jchodera commented:
@ajsilveira : Let me know when I should take a hard look into this, and point me at what I should be looking at closely to see what might be going wrong in that one electrostatic leg!

Commit:
add host-guests and kinase-inhibitors examples

Commit:
fix error when writing pdbs
